platform/security/policy/security-policy-context/src/main/java/org/codice/ddf/security/policy/context/impl/PolicyManager.java

```diff
@@ -235,8 +235,8 @@ public void setPolicies(Map<String, Object> properties) {
       }
 
       for (String attr : attrContextList) {
-        String context = attr.substring(0, attr.indexOf("="));
-        String value = attr.substring(attr.indexOf("=") + 1);
+        String context = attr.substring(0, attr.indexOf('='));
```

Comment:
✏️  Should we just calculate the index of '=' once and use that in both calls?
❓ Are we guarantee that indexOf('=') will never return -1 otherwise this will fail